app/containers/EmojiPicker/EmojiCategory.js

```diff
@@ -64,18 +117,47 @@ export default class EmojiCategory extends React.Component {
 
 	render() {
 		const { emojis } = this.props;
+		const { modalData, isModalVisible } = this.state;
+		const modelStyle = this.getModelStyle();
```

Comment:
This is breaking everything.

Reply:
yaah it seems i changed the name above as you suggested by forgot to change it here ..